readme.md

```diff
@@ -165,12 +165,13 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
   1. Check out the repository, this can be done using [GitHub Desktop](https://desktop.github.com) or [other tools](https://help.github.com/articles/which-remote-url-should-i-use)
   2. Open a new Developer Command Prompt for VS 2022
   3. Navigate to the repository (e.g. `cd C:\GitHub\OpenRCT2`)
-  4. To build the 64-bit version, use `msbuild openrct2.proj /t:build /p:platform=x64`
-
-     To build the 32-bit version, use `msbuild openrct2.proj /t:build /p:platform=Win32`
+  4. To build the x64 version, use `msbuild openrct2.proj /t:build /p:platform=x64`
+     To build the x86 version, use `msbuild openrct2.proj /t:build /p:platform=Win32`
+     To build the Arm64 version, use `msbuild openrct2.proj /t:build /p:platform=arm64`
+     **Note: The file `bin\data\g2.dat` is not generated for Arm64 and must be copied from a x86/x64 build.**
```

Comment:
Why not?

Reply:
There is something about the build system not working on arm64, I cannot remember the reasoning @janisozaur gave on Discord. He has to copy `g2.dat` by hand for each OpenRCT2 release into the Arm64 installer/archive.

Reply:
@janisozaur can we get this fixed so we don't have to instruct users to do this per hand?

Reply:
I found the reply:
> janisozaur: The issue with g2 is we need OpenRCT2 itself to generate the g2. In case of arm64, it is usually an alien architecture and CI cannot generate it.

> janisozaur: I think we discussed extracting g2.dat to another repository

> IntelOrca: Yes all done; Oh I mean the builder is extracted, not g2 files

> janisozaur: It likely won't be done for this release then. Some manual fixup can do that and I need to manually sign the binaries anyway… I might repack it with g2 then

That was back in 2023-04-28.

Reply:
This sounds like a different issue, when manually building the user should be able to run msbuild with the g2 task, it seems as the issue is specific to our CI in where it doesn't package the g2 because it can not run the binary.

Reply:
@ZehMatt I retested this and changed the wording. `g2.dat` can be build on an Arm64 machine, [but you have to do that explicitly](https://github.com/OpenRCT2/OpenRCT2/blob/776674af7ad6261f11c44feecf0cb1eb499cf513/openrct2.proj#L111C50-L111C72).

Cross-compilation is an issue if the machine cannot execute the binaries. E.g. building `g2.dat` for Arm64 on a x64 machine is **not** working. However, building for x86/x64 on an Arm64 machine is (due to emulation).

Reply:
I'm quite frankly not sure if this is the way forward, the condition is probably there so that the CI builds don't fail but for the user who is building it on his own Arm64 machine shouldn't have to worry about this. But then I also think until we get that fixed we can have it in the readme.